### PR TITLE
Convert value of IN and NOT_IN filter to array

### PR DESCRIPTION
**Pull request by codeliner**

### CODE REVIEW

**Review by prolic — APPROVED** [review]

**Comment by prolic** [comment]
Travis failed? Probably only copyright year?

File: tests/unit/Action/LoadStreamTest.php [hunk 1]
```diff
@@ -568,6 +568,130 @@ public function it_returns_only_matched_metadata_and_properties_on_load(): void
...
+        $request->getHeaderLine('Accept')->willReturn('application/vnd.eventstore.atom+json')->shouldBeCalled();
+        $request->getAttribute('streamname')->willReturn('foo')->shouldBeCalled();
+        $request->getAttribute('start')->willReturn('1')->shouldBeCalled();
+        $request->getAttribute('direction')->willReturn('forward')->shouldBeCalled();
+        $request->getAttribute('count')->willReturn('3')->shouldBeCalled();
+        $request->getUri()->willReturn($uri->reveal())->shouldBeCalled();
+        $request->getQueryParams()->willReturn([
+            'meta_0_field' => 'foo',
+            'meta_0_operator' => 'IN',
+            'meta_0_value' => 'bar;baz',
```
Comment (prolic) [inline tests/unit/Action/LoadStreamTest.php:614]: Will array notation not work? Just a question

**Review by prolic — COMMENTED** [review]
Don't forget the LICENSE file, this one is not covered by travis

File: tests/unit/Action/LoadStreamTest.php (hunk 1 above)
Comment (codeliner) [reply tests/unit/Action/LoadStreamTest.php:614]: :thinking: how should such a query look?

`?meta_0_field=_aggregate_version&meta_0_operator=IN&meta_0_value[]=2&meta_0_value[]=3`

**Comment by codeliner** [comment]
damn, newest dependencies seems to crash the build :(

**Comment by codeliner** [comment]
@prolic Can you test if it works on your machine? I don't get errors after `composer update` but travis fails but I don't know why :cry:

File: tests/unit/Action/LoadStreamTest.php (hunk 1 above)
Comment (codeliner) [reply tests/unit/Action/LoadStreamTest.php:614]: ok cool, this query works: `http://localhost:8081/api/v1/stream/event_stream?meta_0_field=_aggregate_version&meta_0_operator=IN&meta_0_value[]=2&meta_0_value[]=3`

Should we close the PR or merge it as a shortcut? Conversion is only done if value is a string, so both ways are valid

**Comment by prolic** [comment]
@codeliner I think we can close this PR. If you need to have a shortcut for this, you can still do it in the frontend. Using `;` is good, but it also limits the possibility to use it somewhere in metadata f.e.

**Comment by prolic** [comment]
About broken tests/ dependencies... I will try to look into it asap, but I have some work on my table right now.

**Pull request closed without merging**